### ISSUE TITLE
[BUG] 1.20 Cannot find module './innerCli'

### ISSUE THREAD

**Issue by Mertsch** [microsoft/playwright-dotnet#2068]
**Context:**
- Playwright Version: 1.20
- Operating System: Windows
- .NET version: .NET 6

```
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module './innerCli'
Require stack:
- C:\Repos\xxx\bin\Debug\net6.0\.playwright\package\lib\cli\cli.js
```

**Describe the bug**

1.20 CLI does not work, 1.19 works just fine

**Comment by mxschmitt** [comment]
Could you try rebuilding your project?

**Comment by Mertsch** [comment]
@mxschmitt I am pretty sure I did, but indeed it fixed the issue. Or reinstalling the package (as I reverted to 1.19).

**Comment by 304NotModified** [comment]
Same issue here. 

D:\SomeWhere [fix-playwright-install-with-ps1 ≡]> pwsh .\bin\release\net5.0\playwright.ps1 install chromium
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module './innerCli'
Require stack:
- D:\SomeWhere\bin\release\net5.0\.playwright\package\lib\cli\cli.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (D:\SomeWhere\bin\release\net5.0\.playwright\package\lib\cli\cli.js:33:3)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'D:\SomeWhere\\bin\\release\\net5.0\\.playwright\\package\\lib\\cli\\cli.js'
  ]
}

**Comment by mxschmitt** [comment]
@304NotModified does a rebuild work for you?

**Comment by 304NotModified** [comment]
Wont help. Tried that indeed.

**Comment by 304NotModified** [comment]
This is under windows

**Comment by mxschmitt** [comment]
what about `dotnet clean` and then `dotnet build?`

**Comment by 304NotModified** [comment]
I will test that

**Comment by chriswelles** [comment]
Same problem here.  `dotnet clean`  and then `dotnetbuild` did not resolve it.  Tried touching the files as referenced in #2069, but that doesn't resolve things either.  Using .NET 6 on windows with latest VS 2022.  I've rolled back to 1.19.1 for the time being.

**Comment by mxschmitt** [comment]
(We'll publish a new version with a fix later today)

**Comment by 304NotModified** [comment]
@mxschmitt 
It the fix already live? What is the version? I cant find it!

**Comment by 304NotModified** [comment]
Is this fixed in 1.20.2?

**Comment by mxschmitt** [comment]
@304NotModified yes it should be fixed.
